src/repo.rs

```diff
@@ -1891,6 +1891,42 @@ impl Repository {
             Ok(())
         }
     }
+
+    /// Add an ignore rule for a repository.
```

Comment:
Could this documentation be expanded a bit to mention that new lines are a llowed, for example?